test/BUILD

```diff
@@ -159,6 +159,17 @@ pipeline_tests(
     filter = "PosTests",
 )
 
+pipeline_tests(
```

Comment:
these tests are already included in `test_corpus`, right?
why would we register & run them twice?